src/Features/Core/CodeFixes/CodeFixService.cs

```diff
@@ -42,14 +42,17 @@ internal partial class CodeFixService : ICodeFixService
 
         private ImmutableDictionary<CodeFixProvider, FixAllProviderInfo> _fixAllProviderMap;
 
+        private readonly IExtensionManager _extensionManager;
+
         [ImportingConstructor]
         public CodeFixService(
             IDiagnosticAnalyzerService service,
+            Workspace workspace,
```

Comment:
how this work? is workspace MEF exported? also, codefix service is not tied to any particular workspace. you can't do this.